### DISCUSSION TITLE
jenkins build failure

### DISCUSSION THREAD

**Discussion by haseeb-abdul9** [learndevopsonline/learndevopsonline#1860] [Q&A]
trying to run terraform code using jenkins but the build fails with multiple errors
jenkins server : 34.239.233.141 | 172.31.8.215
jenkins user & pass : admin | admin123

workstation server : 54.147.77.31 | 172.31.7.30

_repos_
roboshop_shell: https://github.com/haseeb-abdul9/roboshop_shell.git
roboshop_terraform: https://github.com/haseeb-abdul9/roboshop_terraform.git

**Comment by chaithanya1812** [comment]
Hi @haseeb-abdul9,
check once jenkins server is not reachable..

**Reply by haseeb-abdul9** [reply]
Should be reachable now 
Thanks

**Reply by chaithanya1812** [reply]
will you share the  jenkins new one public-ip ..

**Reply by haseeb-abdul9** [reply]
3.239.227.41
